src/pkcs1_5.c

```diff
@@ -338,8 +338,6 @@ static int rimage_verify(EVP_PKEY *privkey, enum manver ver,struct hash_context
 		fprintf(stderr, "error: verify %s\n", err_buf);
 	}
 
-	break;
-
 out:
```

Comment:
why do you remove the break? It doesn't seem to belong to this patch if you really want to remove it.

Reply:
the break is in the middle of a function outside switch statement... so probably some copy/paste error, probably better just to remove. I mean this break was introduced in the same patch @aiChaoSONG is trying to fix here.

Reply:
ok, so we are fixing a copy/paste error in the previous patch .